types/context.ts

```diff
@@ -0,0 +1,12 @@
+import { Proposal } from 'types/proposal';
+import { DAO } from 'types/dao';
+
+export type UserPermissions = {
+  isCanCreateProposals: boolean;
+};
+
+export type DaoContext = {
```

Comment:
`DaoContext` looks a bit confusing, cause I would actually expect react context for it, but we are not using context. Maybe, `DaoInfo` or `DaoWithPermission` or any other proper name?

Reply:
But context is not only about react :) its some general information about the Dao